tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -289,13 +289,11 @@ these properties, and if you don't write to them, Godot will optimize away the c
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
 | in int **VIEW_RIGHT**                  |                                                                                                  |
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
-| sampler2D **SCREEN_TEXTURE**           | Built-in Texture for reading from the screen. Mipmaps contain increasingly blurred copies.       |
+| sampler2D **SCREEN_TEXTURE**           | Deprecated in Godot 4. Use a ``sampler2D`` with ``hint_screen_texture`` instead.                 |
```

Comment:
```suggestion
| sampler2D **SCREEN_TEXTURE**           | Removed in Godot 4. Use a ``sampler2D`` with ``hint_screen_texture`` instead.              |
```
"Deprecated" is usually used for something which still works, but might be removed in the future. Here this is not defined at all anymore.